### PR TITLE
improve SEM data library example

### LINKED ISSUE #2337: [Bug]: Data library SEM file without .jasp file
### JASP Version

0.19 beta

### Bug Description

The data sets "social participation" and "grand point average" only come with a csv file in jasps data library

### Expected Behaviour

If I open those files with a click on the green button I expect, that a .jasp file with results and explanations is opened.

### PR DESCRIPTION
**Pull request by juliuspfadt**
fixes https://github.com/jasp-stats/jasp-issues/issues/2337
fixes https://github.com/jasp-stats/INTERNAL-jasp/issues/2486

The social participation example does not really help, for the grand point average I made an LGCM example.

### CODE REVIEW

File: Resources/Data Sets/index.json [hunk 1]
```diff
@@ -860,14 +860,14 @@
...
           "associated_datafile": "Political Democracy.csv"
         },
         {
           "name": "Grand Point Average",
           "path": "Grand Point Average.jasp",
-          "description": "Grand Point Average of 200 college students across 6 semesters, along with information on gender, high school GPA and whether they were admitted to their university of choice. <br><br> The JASP file currently does not contain any analysis. <br><br> <i>Data from Hox, Moerbeek, and Van de Schoot (2017).<\/i>",
+          "description": "Grand Point Average of 200 college students across 6 semesters, along with information on gender, high school GPA, whether they were admitted to their university of choice, and job working hours across 6 semesters. <br><br> The JASP file demonstrates the use of latent growth curve modelling (LGCM). <br><br> <i>Data from Hox, Moerbeek, and Van de Schoot (2017).<\/i>",
           "kind": "file",
           "associated_datafile": "Grand Point Average.csv"
         },
```
Comment (JorisGoosen) [inline Resources/Data Sets/index.json]: Shouldnt you  remove "Social Participation" below here from the index.json(s)?

**Review by JorisGoosen — CHANGES_REQUESTED** [review]

**Comment by JorisGoosen** [comment]
If I open and refresh the GPA jaspfile I get this: 
image
Seems related to this: https://github.com/jasp-stats/jaspSem/commit/c5044eed9c3d5a5bc6946b66e8de773740c1815c
This is on my local build of your branch rebased on upstream/development

The other example seems to work fine.

**Comment by juliuspfadt** [comment]
ja indeed, I checked it with the fix installed, and it works. I might tell Simon to review the PR for the time being.

**Comment by JorisGoosen** [comment]
Ok nice that did it!

**Review by JorisGoosen — APPROVED** [review]

**Pull request merged**